src/Domain/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Codidact.Domain.Extensions
+{
+    public static class StringExtensions
+    {
+        /// <summary>
+        /// Converts a string to snake_case
+        /// </summary>
+        /// <param name="input">The string to convert</param>
+        /// <returns></returns>
+        public static string ToSnakeCase(this string input)
+        {
+            if (string.IsNullOrEmpty(input)) { return input; }
```

Comment:
Use multiple lines for if-statements.

Reply:
Is there a style guide for coding?

Reply:
Not yet for C# its being written (in progress)
But for this case specifically @ArtOfCode- mentioned this in discord and in general yes, this shouldn't be in one line.